### DISCUSSION TITLE
Using Automatic1111 in the WebUI

### DISCUSSION THREAD

**Discussion by nozem79** [open-webui/open-webui#935] [General]
Hi,
I now successfully integrated Automatic1111 in Open WebUI, but ofcource I not know how to use is, is there a specific model of prompt to use it?

#probablyastupidquestion

**Comment by justinh-rahb** [comment]
No specific model, though there are txt2img-generator specific models out there (none supported directly in Ollama that I know of). You can check out the guide on the docs site:

https://docs.openwebui.com/tutorial/images

**Reply by nozem79** [reply]
A bit buggy, but it works :-)
